webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/services/reports/ComputeNodeStatusReportService.scala

```diff
@@ -122,6 +126,24 @@ trait ComputeNodeStatusReportService extends InvalidateCache[CacheComplianceQueu
 
 }
 
+trait NodeStatusReportUpdateHook {
+  def name:     String
+  def onUpdate: Iterable[(NodeId, NodeStatusReport)] => UIO[Unit]
+}
```

Comment:
A class to handle result of saved NodeStatusReport changed - with below the only implem for now, for scoring service.